client/packages/invoices/src/OutboundShipment/DetailView/ContentArea.tsx

```diff
@@ -112,22 +112,24 @@ export const ContentAreaComponent: FC<ContentAreaProps> = ({
           />
         </Box>
       )}
-      <DataTable
-        id="outbound-detail"
-        onRowClick={onRowClick}
-        ExpandContent={Expand}
-        columns={columns}
-        data={rows}
-        enableColumnSelection
-        noDataElement={
-          <NothingHere
-            body={t('error.no-outbound-items')}
-            onCreate={isDisabled ? undefined : () => onAddItem()}
-            buttonText={t('button.add-item')}
-          />
-        }
-        isRowAnimated={true}
-      />
+      <Box flex={1} style={{ overflowY: 'auto' }}>
```

Comment:
using flex to get the container to expand to fill the vertical space, adding `overflowY` to get the windowing working on the datatable and adding this extra container so that the `100%` height on the datatable is of this container not the one which also contains the group by row.